### PR TITLE
Upgrade Authorization to latest swagger spec

### PR DESCRIPTION
**Pull request by ArthurMa1978**

### CODE REVIEW

File: sdk/authorization/Azure.ResourceManager.Authorization/src/Generated/RoleAssignmentData.cs [hunk 1]
```diff
@@ -24,13 +25,57 @@ internal RoleAssignmentData()
...
+        /// <summary> Time it was created. </summary>
+        public DateTimeOffset? CreatedOn { get; }
+        /// <summary> Time it was updated. </summary>
+        public DateTimeOffset? UpdatedOn { get; }
+        /// <summary> Id of the user who created the assignment. </summary>
+        public string CreatedBy { get; }
+        /// <summary> Id of the user who updated the assignment. </summary>
+        public string UpdatedBy { get; }
+        /// <summary> Id of the delegated managed identity resource. </summary>
+        public string DelegatedManagedIdentityResourceId { get; }
```
Comment (Yao725) [inline sdk/authorization/Azure.ResourceManager.Authorization/src/Generated/RoleAssignmentData.cs:79]: Should it be an `ResourceIdentifier`?

File: sdk/authorization/Azure.ResourceManager.Authorization/src/Generated/Models/RoleAssignmentCreateOrUpdateContent.cs [hunk 2]
```diff
@@ -6,26 +6,51 @@
...
+        /// <summary> Time it was created. </summary>
+        public DateTimeOffset? CreatedOn { get; }
+        /// <summary> Time it was updated. </summary>
+        public DateTimeOffset? UpdatedOn { get; }
+        /// <summary> Id of the user who created the assignment. </summary>
+        public string CreatedBy { get; }
+        /// <summary> Id of the user who updated the assignment. </summary>
+        public string UpdatedBy { get; }
+        /// <summary> Id of the delegated managed identity resource. </summary>
+        public string DelegatedManagedIdentityResourceId { get; set; }
```
Comment (Yao725) [inline sdk/authorization/Azure.ResourceManager.Authorization/src/Generated/Models/RoleAssignmentCreateOrUpdateContent.cs:54]: Should it be an `ResourceIdentifier`?

File: sdk/storagecache/Azure.ResourceManager.StorageCache/package.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (Yao725) [inline sdk/storagecache/Azure.ResourceManager.StorageCache/package.json:1]: Why do we need this file and another `package-lock.json` file in StorageCache?

File: sdk/authorization/Azure.ResourceManager.Authorization/src/Generated/RoleAssignmentData.cs (hunk 1 above)
Comment (ArthurMa1978) [reply sdk/authorization/Azure.ResourceManager.Authorization/src/Generated/RoleAssignmentData.cs:79]: Pending on service team's confrm, from test it's more like a raw string

File: sdk/authorization/Azure.ResourceManager.Authorization/src/Generated/Models/RoleAssignmentCreateOrUpdateContent.cs (hunk 2 above)
Comment (ArthurMa1978) [reply sdk/authorization/Azure.ResourceManager.Authorization/src/Generated/Models/RoleAssignmentCreateOrUpdateContent.cs:54]: Pending on service team's confrm, from test it's more like a raw string

File: sdk/storagecache/Azure.ResourceManager.StorageCache/package.json (hunk 3 above)
Comment (ArthurMa1978) [reply sdk/storagecache/Azure.ResourceManager.StorageCache/package.json:1]: hmm... it's checked in by mistake

Comment (ArthurMa1978) [reply sdk/storagecache/Azure.ResourceManager.StorageCache/package.json]: fixed

**Review by Yao725 — APPROVED** [review]

**Pull request merged**